src/pages/components/tile/usage.mdx

```diff
@@ -61,13 +71,25 @@ ecosystem below:
 ### When to use
 
 Tiles are reusable components that provide shortcuts to building cards and other
-modules. Here are some common use cases for when to use tiles:
+modules. Use tiles as part of the user's main workstream. Here are some common
+use cases for when to use tiles:
 
 - To contain related groupings of information or actions
 - To guide users to take actions or navigate
 - To present options for single or multiple selections
 - To hide or show large amounts of content
 
+### When not to use
+
+Tiles reside in the same plane as the background layer—they do not have
+**elevation**. Tiles organize essential information and have the same visual
+hierarchy as content within the same page.
+
+Do not add a drop shadow to tiles and use them on the screen to reveal secondary
+information, actions, or notifications. Use [modals](/components/modal/usage),
+[popover](/components/popover/usage), and [dialog](/components/dialog/usage)
+which have elevation and are appropriate for this function instead.
```

Comment:
we do not currently have a component page for dialog in website so this dialog link is returning 404: 
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/40550942/218191610-0a25099f-39d1-4672-b1ca-98a56e2b2a78.png">